src/gc.h

```diff
@@ -11,20 +11,24 @@
 
 #define V7_DISABLE_GC
 
-#define MARK(p) (* (uintptr_t *) &(p) |= 1)
+#define MARK(p) (* (unsigned long *) &(p) |= 1)
 
 /* call only on already marked values */
-#define UNMARK(p) (* (uintptr_t *) &(p) &= ~1)
+#define UNMARK(p) (* (unsigned long *) &(p) &= ~1)
 
-#define MARKED(p) ((uintptr_t) (p) & 1)
+#define MARKED(p) (((unsigned long) (void *) (p)) & 1)
 
 struct gc_tmp_frame {
   struct v7 *v7;
   size_t pos;
 };
 
+#ifdef _WIN32
+#define GC_TMP_FRAME(v) struct gc_tmp_frame v = new_tmp_frame(v7);
```

Comment:
This is not a functional replacement.

We should replace this with some nop code and disable GC on windows until we handle tmp frame cleanup manually in all functions.